### PR TITLE
Feat: Implementation of scripts to interact with the contract

### PR DESCRIPTION
**Pull request by Guboehm**

### CODE REVIEW

File: scripts/getTask.ts [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
 
-    const contract = new ethers.Contract("0xec20dcbf0380f1c9856ee345af41f62ee45a95a1", abi.abi, signer);
-
-    const getTaskResponse = await contract.getTask(0);
-
-
-    console.log('GetTaskResponse = ', getTaskResponse);
+    const contract = new ethers.Contract(`${CONTRACT_ADDRESS}`, abi.abi, signer);
 
+    for (let i = 0; i <= 50; i++) {
```
Comment (0xneves) [inline scripts/getTask.ts:11]: The getTask shouldn't have a for loop. We currently have a script for multiCall precisely to cover this need.

File: scripts/startTask.ts
Comment (0xneves) [inline scripts/startTask.ts:1]: Wrong tabs on. line 12,13,14

File: scripts/setRole.ts
Comment (0xneves) [inline scripts/setRole.ts:1]: Wrong tabs on line 12,13,14

File: scripts/setQuorum.ts
Comment (0xneves) [inline scripts/setQuorum.ts:1]: Wrong tabs on line 12,13,14

File: scripts/setOperator.ts
Comment (0xneves) [inline scripts/setOperator.ts:1]: Wrong tabs on line 12,13,14

File: scripts/reviewTask.ts
Comment (0xneves) [inline scripts/reviewTask.ts:1]: Wrong tabs on line 12,13,14

File: scripts/getTask.ts
Comment (0xneves) [inline scripts/getTask.ts:1]: The getTask shouldn't have a for loop. We currently have a script for multiCall precisely to cover this need.

File: scripts/getMulticallTask.ts
Comment (0xneves) [inline scripts/getMulticallTask.ts:1]: Wrong tabs in lines 7,9?

File: scripts/deploy.ts
Comment (0xneves) [inline scripts/deploy.ts:1]: Wrong tabs on lines 26

File: scripts/createTask.ts
Comment (0xneves) [inline scripts/createTask.ts:1]: Wrong tabs

File: scripts/completeTask.ts
Comment (0xneves) [inline scripts/completeTask.ts:1]: Wrong tabs

File: scripts/cancelTask.ts
Comment (0xneves) [inline scripts/cancelTask.ts:1]: Wrong tabs across all files

**Review by 0xneves — CHANGES_REQUESTED** [review]
Wrong tabs in almost all files and getTask is using for loop, it shouldn't

**Comment by 0xneves** [comment]
@Guboehm Could you make a random value for the reward field, something around 0.001 and 0.00001. This way we can actually test with the mocks and also have randomness between them besides the titles

File: scripts/getTask.ts (hunk 1 above)
Comment (Guboehm) [reply scripts/getTask.ts:11]: ok

**Pull request merged**